docs/General/Manuals/client-manual.rst

```diff
@@ -311,6 +311,51 @@ scenario file in an alternate location. Clicking on :guilabel:`Cancel` will brin
 Options
 -------
 
+When you click on the :guilabel:`Options` button, a dialog box will appear as in Figure 8 below. This dialog
+allows you to set a myriad of options that effect the look at feel of the client when you are playing a game.
+
+The local client options dialog box can also be opened while in a game via the
+:menuselection:`Game --> Local Options` menu.
+
+.. figure:: ../../_static/images/gui-elements/local-options.png
+    :height: 400px
+    :align: center
+    :alt: Freeciv21 Local Options Dialog
+    :figclass: align-center
+
+    Figure 8: Local Options Dialog
+
+
+The local options dialog is broken down into the following tabs:
+
+* :guilabel:`Network`: On this tab you can save your preferred username, server and port information to be
+  used during `Connect to Network Game`_.
+* :guilabel:`Sound`: On this tab you can set everything related to in game sound and music.
+* :guilabel:`Interface`: On this tab you can set anything related to how you interact with the interface of
+  client while playing a game.
+* :guilabel:`Graphics`: On this tab you can change the look and feel with a different theme. NightStalker is
+  the default theme out of the box. You can also set the default tileset to use under different map styles as
+  well as various things you may or may not want the client to paint (draw) on the screen.
+* :guilabel:`Overview`: On this tab you can enable/disable varying layers of the map.
```

Comment:
I'm not sure the user understands what a layer is by this point. Maybe say something like disable various elements/aspects of the map ? Or maybe explain layers in the mapview section and link it here. 

Side note: I think we need to fix this, the separation between interface and overview makes little sense.

Reply:
@pranavtifr would you please open an issue on this side note fix? I think you can describe it better than I can.

Reply:
I changed the text to say "features" instead of "layers"